tests/smoke/.makim-scheduler.yaml

```diff
@@ -17,19 +35,19 @@ groups:
       test-all:
         hooks:
           pre-run:
-            - task: test-echo
-            - task: test-date
-            - task: test-sleep
+            - task: test.test-echo
+            - task: test.test-date
+            - task: test.test-sleep
 
 scheduler:
   test_basic_echo:
     schedule: "*/5 * * * *"
-    task: test.test-echo
+    task: default.test-echo
```

Comment:
it is not necessary to change from default to test
the same applies to the other ones as well.